portfolio/src/main/webapp/style/format.css

```diff
@@ -398,6 +410,61 @@ a {
   clear: both;
 }
 
+<<<<<<< HEAD
+#map-container {
+  background-color: transparent;
+  display: flex;
+  flex: 1.5;
+  align-items: center;
+}
+
+#map {
+  height: 50vh;
+  width: 35vw;
+}
+
+#about-description {
+  flex: 2;
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+  color: black;
+  margin: 5px 5px;
+}
+
+#about-pictures {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+  color: black;
+}
+
+.about-header {
+  border-bottom: 3px solid black;
+  display: flex;
+  align-items: center;
+  font-size: 45px;
+  flex: 1;
+  text-align: center;
+  margin: 10px 10px;
+}
+
+.about-body {
+  border-bottom: 3px solid black;
+  flex: 2;
+}
+
+.about-me {
+  font-size: 25px;
+  text-align: center;
+  margin: 10px 10px;
+}
+
+.about-under {
+  flex: 2;
+}
+=======
```

Comment:
same here (ofc check that the merge conflict was resolved properly)

Reply:
done